packages/engine/Source/Scene/CreditDisplay.js

```diff
@@ -450,24 +477,58 @@ CreditDisplay.prototype.addCredit = function (credit) {
  * Adds credits that will persist until they are removed
  *
  * @param {Credit} credit The credit to added to defaults
+ *
+ * @deprecated
  */
 CreditDisplay.prototype.addDefaultCredit = function (credit) {
   //>>includeStart('debug', pragmas.debug);
   Check.defined("credit", credit);
   //>>includeEnd('debug');
 
+  deprecationWarning(
+    "CreditDisplay.addDefaultCredit",
+    "CreditDisplay.addDefaultCredit was deprecated in CesiumJS 1.105.  It will be in CesiumJS 1.107.  Use CreditDisplay.addStaticCredit instead."
+  );
+
   const defaultCredits = this._defaultCredits;
   if (!contains(defaultCredits, credit)) {
+    credit.showOnScreen = true;
     defaultCredits.push(credit);
   }
 };
 
 /**
- * Removes a default credit
+ * Adds a {@link Credit} that will show on screen or in the lightbox until removed with {@link CreditDisplay.removeStaticCredit}.
  *
- * @param {Credit} credit The credit to be removed from defaults
+ * @param {Credit} credit The credit to added
+ *
+ * @example
+ * // Add a credit with a tooltip, image and link to display onscreen
+ * const credit = new Cesium.Credit(`<a href="https://cesium.com/" target="_blank"><img src="/images/cesium_logo.png" title="Cesium"/></a>`, true);
+ * viewer.creditDisplay.addStaticCredit(credit);
+ *
+ * @example
+ * // Add a credit with a plaintext link to display in the lightbox
+ * const credit = new Cesium.Credit('<a href="https://cesium.com/" target="_blank">Cesium</a>');
+ * viewer.creditDisplay.addStaticCredit(credit);
  */
-CreditDisplay.prototype.removeDefaultCredit = function (credit) {
+CreditDisplay.prototype.addStaticCredit = function (credit) {
+  //>>includeStart('debug', pragmas.debug);
+  Check.defined("credit", credit);
+  //>>includeEnd('debug');
+
+  const defaultCredits = this._defaultCredits;
```

Comment:
I like the new public-facing API naming. For the sake of future maintainers, can we make the internal naming consistent?  `this._defaultCredits` could be `this._staticCredits`